resources/assets/lib/utils/score-helper.ts

```diff
@@ -97,6 +97,8 @@ export const modeAttributesMap: Record<GameMode, AttributeData[]> = {
 export function modName(acronym: string): string {
   if (acronym === 'V2') {
     return 'Score V2';
+  } else if (acronym === 'NM') {
```

Comment:
looking again, those additional names should just be added when generating the `mod-names.json` file...